usagov/vote-gov
# Vote-109 -> Russian accessibility page

link to ticket: [Vote-109](https://cm-jira.usa.gov/browse/VOTE-109)

to test: 

1. pull down branch 
2. run `npm run start`
3. visit local site and change to Russian and select the accessibility page and ensure that page loads and not regression
